File: Eksamensopgaven/ExamProject/app/src/main/java/examproject/group22/roominator/Models/Apartment.java
package examproject.group22.roominator.Models;

import java.io.Serializable;
import java.util.ArrayList;

/**
 * Created by Mathias on 08-Oct-16.
 */

public class Apartment implements Serializable{

    public int id;
    public final String name;
    public final String password;
    public ArrayList<GroceryItem> groceries;
    public ArrayList<User> users;

    public Apartment(String name, String password)
    {
        this.name = name;
        this.password = password;
    }
}

File: RoomienatorWeb/RoomienatorWeb/Models/Apartment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace RoomienatorWeb.Models
{
    public class Apartment
    {
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public string Pass { get; set; }

        public virtual ICollection<User> Users { get; set; }
        public virtual ICollection<GroceryItem> GroceryItems { get; set; }
    }
}